lua/entities/gmod_wire_expression2/core/color.lua

```diff
@@ -94,6 +81,7 @@ e2function void entity:setRenderMode(mode)
 	if this:IsPlayer() then return end
 
 	this:SetRenderMode(mode)
+	duplicator.StoreEntityModifier(this, "colour", { RenderMode = mode })
```

Comment:
This won't dupe correctly, we need to save the color here too:

```Lua
Prop:setColor(255, 255, 255, 64)
Prop:setRenderMode(_RENDERMODE_GLOW)
```

Reply:
It will work fine. The modifier function checks if color is nil or not.

Reply:
Also modifiers are merged so existing Color key will remain.

Reply:
Ah, cool